lib/condensation/filters/hmac_sha256.rb

```diff
@@ -0,0 +1,10 @@
+module Condensation
+  module Filters
+    module HmacSha256
+      def hmac_sha256(input, secret_key)
+        return if input.nil?
+        OpenSSL::HMAC.hexdigest("SHA256", secret_key, input)
+      end
+    end
+  end
+end
```

Comment:
We should explicitly require the openssl lib:

```
irb(main):001:0> OpenSSL::HMAC
NameError: uninitialized constant OpenSSL
	from (irb):1
	from /Users/derrickreimer/.rbenv/versions/2.3.1/bin/irb:11:in `<main>'
irb(main):002:0> require 'openssl'
=> true
irb(main):003:0> OpenSSL::HMAC
=> OpenSSL::HMAC
```

A `require "openssl"` at the top of the file should do the trick.

Reply:
Thanks @djreimer Updated.